packages/just-scripts/src/tasks/webpackCLITask.ts

```diff
@@ -0,0 +1,57 @@
+import { logger, TaskFunction } from "just-task";
+import { tryRequire } from "../tryRequire";
+
+/**
+ * webpackCliInitTask - task for webpack-cli init command
+ *
+ * @param  {string} customScaffold? - to pass any webpack-scaffold
+ * @param  {Boolean=false} auto - to pass the --auto flag, which will generate a default webpack.config.js
+ * @returns TaskFunction
+ */
+export function webpackCliInitTask(
+  customScaffold?: string,
+  auto: Boolean = false
+): TaskFunction {
+  return async function webpackCli() {
+    const init = tryRequire("@webpack-cli/init").default;
+    if (!init) {
+      logger.warn(
+        "webpack-cli init requires three dependencies: @webpack-cli/init (preferred - as a devDependency )"
+      );
+      return;
+    }
+    logger.info(`Running Webpack-cli init `);
+    if (typeof customScaffold === "undefined") {
+      if (auto) {
+        return new Promise((resolve, reject) => {
+          try {
+            init(null, null, null, "--auto");
```

Comment:
This looks like a synchronous thing, you don't really need to return Promise in that case, right?

Reply:
the generator running seems async . I will take a bit look at this and will refactor the commit then .
May I think its sync , we done need promise then !

Reply:
ping

Reply:
Sorry for the delay,  I will try to resolve this by this weekend :+1:

Reply:
@kenotron Done 👍 
Changed it to sync !.